packages/loader/driver-utils/src/network.ts

```diff
@@ -129,11 +129,11 @@ export const createWriteError = (errorMessage: string) =>
 export function createGenericNetworkError(
     errorMessage: string,
     canRetry: boolean,
-    retryAfterMs?: number,
+    retryAfterSeconds?: number,
     props?: ITelemetryProperties,
 ): ThrottlingError | GenericNetworkError {
-    if (retryAfterMs !== undefined && canRetry) {
-        return new ThrottlingError(errorMessage, retryAfterMs / 1000, props);
+    if (retryAfterSeconds !== undefined && canRetry) {
+        return new ThrottlingError(errorMessage, retryAfterSeconds, props);
```

Comment:
It feels better as long as we don't care about this granularity. But if we only care about consistency, probably changing ThrottlingError to use milliseconds would make the change have a smaller impact.

Reply:
We can, but such change has implications (i.e. getRetryDelayFromError and impact on back compat between layers).
Plus all telemetry would be in milliseconds (as it gets logged).
It's all doable, question is - what we want :)

Reply:
Honestly I'd use something like a `Duration` class and make this less opinionated.